mycore-pi/src/main/java/org/mycore/pi/urn/rest/MCRDNBURNRestClient.java

```diff
@@ -161,6 +164,36 @@ public Optional<Date> register(MCRPIRegistrationInfo urn) {
         return Optional.empty();
     }
 
+    public Optional<JsonObject> getRegistrationInfo(MCRPIRegistrationInfo urn) {
+        String identifier = urn.getIdentifier();
+        return getRegistrationInfo(identifier);
+    }
+
+    public static Optional<JsonObject> getRegistrationInfo(String identifier) {
+        MCRDNBURNRestClient mcrurnClient = new MCRDNBURNRestClient(
```

Comment:
If you make some other methods static you can remove the instance of the class and the weird constructor call is not needed.